Containerfile.rag

```diff
@@ -1,21 +1,18 @@
 # vim: set filetype=dockerfile
-
-# port, commands, and other details inherited from the base image
 ARG OLS_VERSION=latest
+ARG OCP_VERSION=4.15
+ARG LIGHTSPEED_RAG_CONTENT_DIGEST=sha256:eb3472dd675cc79b02e4d15b53b12c9c365d6c2335a6026f55de443dd95c902f
+
+FROM quay.io/openshift/lightspeed-rag-content@${LIGHTSPEED_RAG_CONTENT_DIGEST} as lightspeed-rag-content
+ARG OLS_VERSION
+
 FROM quay.io/openshift/lightspeed-service-api:${OLS_VERSION}
+ARG OLS_VERSION
+ARG OCP_VERSION
 
-ARG RAG_INDEX="https://github.com/ilan-pinto/lightspeed-rag-documents/releases/latest/download/local.zip"
 ARG APP_ROOT=/app-root
+WORKDIR ${APP_ROOT}
 USER 0
 
-RUN microdnf install -y unzip wget \
-    && mkdir -p vector-db \
-    && cd vector-db \
-    && wget -q $RAG_INDEX \
-    && unzip -qq local.zip \
-    && mv local ocp-product-docs \
-    && rm -f local.zip \
-    && microdnf remove -y unzip wget
-
-COPY scripts/download_embeddings_model.py .
-RUN python3.11 download_embeddings_model.py -l embeddings_model -r BAAI/bge-base-en && rm download_embeddings_model.py
+COPY --from=lightspeed-rag-content /rag/vector_db/ocp_product_docs/${OCP_VERSION} ./vector_db/ocp_product_docs/${OCP_VERSION}
```

Comment:
how are we going to support multiple ocp versions like this?  ultimately the emebddings image should/will contain embeddings for multiple OCP docs versions, and we should just copy them all into our OLS image.

Reply:
Are you saying to just drop any mention of of OCP_VERSION from Container.rag and
```
COPY --from=lightspeed-rag-content /rag/vector_db/ocp_product_docs ./vector_db/ocp_product_docs
```
?

Reply:
yeah

Reply:
Done